components/AudioSettings/AudioSettings.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useRef } from "react";
+import MuteButton from "./MuteButton";
+import PlaySound from "./PlaySound";
+import DecreaseVolume from "./DecreaseVolume";
+
+export default function AudioSettings({ page, experience, audioTrigger }) {
```

Comment:
I guess the `page` prop is passed but not used; consider removing it if unnecessary 🙂

Reply:
Should have cleaned up the mess before PR 😅 Thank you!